misc/python/materialize/cli/ci_logged_errors_detect.py

```diff
@@ -55,6 +55,8 @@
     (?!.*timely\ communication\ error:)
     # Expected once compute cluster has panicked, only happens in CI
     (?!.*aborting\ because\ propagate_crashes\ is\ enabled)
+    # Emitted by webhook source tests that explicitly panic the validation.
+    (?!.*panic\ while\ validating\ webhook\ request')
```

Comment:
@ParkMyCar: Are you sure that this is the right string? I think we need to match this one: `thread 'tokio:work-45' panicked at '{accept=>*/*,content-length=>3,host=>materialized:6876}'`

See build: https://buildkite.com/materialize/nightlies/builds/2868